package.json

```diff
@@ -50,7 +50,7 @@
     "ajv": "^6.12.4",
     "debug": "^4.1.1",
     "espree": "^7.3.0",
-    "globals": "^12.1.0",
+    "globals": "^13.6.0",
```

Comment:
globals is up to 13.9 now, which is still compatible with the version defined in eslint